admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.js

```diff
@@ -71,8 +71,8 @@ export default {
   productsTablePaginationLink: '.page-item:not(.d-none):not(#orderProductsTablePaginationNext):not(#orderProductsTablePaginationPrev) .page-link',
   productsTablePaginationActive: '#orderProductsTablePagination .page-item.active span',
   productsTablePaginationTemplate: '#orderProductsTablePagination .page-item.d-none',
-  productsTableRow: (productId) => `#orderProduct_${productId}`,
-  productsTableRowEdited: (productId) => `#editOrderProduct_${productId}`,
+  productsTableRow: productId => `#orderProduct_${productId}`,
```

Comment:
I believe eslint requires those `()` and removing them will conflict with the rules on develop
@NeOMakinG what do you think?

Reply:
I think I've deleted it because the eslinter underlined those two lines.

Reply:
Ok it's weird because in function callbacks the `()` are required :P Maybe it depends on the eslint config? Is it the same as the one in develop?
But anyway if eslint is fine with it no problem for me

Reply:
They're required on develop, it will generate conflict on the merge of 1.7.7.x into develop.

I think your IDE use a basic configuration (like eslint:recommended) or airbnb, because he can't find a configuration on the project (of course it's not existing on 1.7.7.x), maybe you should configure you'r IDE to use our configurastion even if there are no .eslintrc files

Reply:
rolled back